test/integration/fn_mount_cmd.go

```diff
@@ -62,10 +62,10 @@ func validateMountCmd(ctx context.Context, t *testing.T, profile string) {
 		t.Fatalf("Unexpected error while creating tempDir: %v", err)
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, Minutes(10))
```

Comment:
Move unrelated context changes into another PR. There is too much going on in this PR to decipher why this change is being done.